operator/pkg/helmreconciler/reconciler.go

```diff
@@ -252,19 +237,14 @@ func (h *HelmReconciler) processRecursive(manifests name.ManifestMap) *v1alpha1.
 
 // CheckSSAEnabled is a helper function to check whether ServerSideApply should be used when applying manifests.
 func (h *HelmReconciler) CheckSSAEnabled() bool {
-	if h.restConfig != nil {
-		// check k8s minor version
-		k8sVer, err := k8sversion.GetKubernetesVersion(h.restConfig)
-		if err != nil {
-			scope.Errorf("failed to get k8s version: %s", err)
-		}
+	if h.kubeClient != nil {
 		// There is a mutatingwebhook in gke that would corrupt the managedFields, which is fixed in k8s 1.18.
 		// See: https://github.com/kubernetes/kubernetes/issues/96351
-		if k8sVer >= 18 {
+		if kube.IsAtLeastVersion(h.kubeClient, 18) {
```

Comment:
nit: We are going to only support 1.19+ for Istio 1.13, correct? This can be changed once that happens.